R/vault.R

```diff
@@ -0,0 +1,33 @@
+#' Use configured vault to resolve secrets.
+#'
+#' This is copied from orderly.
+#'
+#' @param x List of properties some of which may be vault secrets of the form
+#' VAULT:<path to secret>:<key to get>
+#' @param config Cnfig containing details of the vault server to be accessed.
+#'
+#' @return List of properties with any secrets resolved secret.
+#'
+#' @keywords internal
+#'
+resolve_secrets <- function(x, config) {
```

Comment:
If we're using this in both `orderly` and in `dettl`, do you think it should go into `vaultr` perhaps?

Reply:
I think that would be a good idea. It it identical between `orderly` and `dettl`.